lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -246,6 +250,7 @@ class RadarChartPainter extends BaseChartPainter<RadarChartData> {
 
   RadarTouchedSpot? handleTouch(
       PointerEvent touchInput, Size size, PaintHolder<RadarChartData> holder) {
+    _triggerTitleTap(touchInput.down, touchInput.localPosition, holder);
```

Comment:
`handleTouch` function gets called in every touch event (such as move touch).
So this is not the best way to handle this.
We need to change our approach to handle this kind of custom touch. (I know you need it in your use-case) But we cannot handle this as an instant solution (because we need to consider our long-term approach).
So please revert these codes for now, and open an issue to follow it up.

Thanks!

Reply:
@imaNNeoFighT please take a look at the latest changes

Reply:
@imaNNeoFighT it'd be great if you check the latest changes

Reply:
I don't find where you are checking _TransformedPointerUpEvent?

There is only one if block which is :
```
if (isDownEvent || _titlesPosition.isEmpty) return;
```

Reply:
> I don't find where you are checking _TransformedPointerUpEvent?
> 
> There is only one if block which is :
> 
> ```
> if (isDownEvent || _titlesPosition.isEmpty) return;
> ```

Yes, actually it is, @imaNNeoFighT

Reply:
@imaNNeoFighT , can I leave it for merge or should I remove it?

Reply:
Please remove `_triggerTitleTap` to speed up the merge process.
We can talk about it separately.

Sorry for my delay. I was busy.